zuul.d/project-templates.yaml

```diff
@@ -453,24 +453,13 @@
             required-projects:
               - name: github.com/ansible-collections/cloud.common
               - name: github.com/ansible-collections/community.vmware
-        - ansible-test-sanity-docker-devel
         - ansible-test-sanity-docker-milestone
```

Comment:
Why don't we add milestone branch to the GHA?

Reply:
I'm not an expert on this GHA stuff, but as far as I understand it's not that easy to make the GHA sanity tests non-voting on GHA. Having this as non-voting won't prevent us from merging PRs but would also give us some information that we might run into problems in the future.

On the other hand, I think we also can remove this test for now even without moving it to GHA. If you want me to, I'll update my PR accordingly. Or you do, because it's time fore me to go to bed...

Reply:
I've pushed a new commit removing the milestone sanity tests. We can add it later to the GHA workflows in the collection if we want, but let's remove it from Zuul.